state/txn.go

```diff
@@ -8,13 +8,18 @@ import (
 
 	"github.com/0xPolygon/polygon-edge/chain"
 	"github.com/0xPolygon/polygon-edge/crypto"
-	"github.com/0xPolygon/polygon-edge/helper/keccak"
 	"github.com/0xPolygon/polygon-edge/state/runtime"
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
 var emptyStateHash = types.StringToHash("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
 
+type readSnapshot interface {
+	GetStorage(addr types.Address, root types.Hash, key types.Hash) types.Hash
```

Comment:
Could it be just 

```
Storage(...)
Account(...)
Code(...)
```

Reply:
Aren't those names less descriptive of what the functions do?

Reply:
Agree with @JekaMas, why use `Get` prefix?

Reply:
Do you feel that `Storage` alone represents what the method does?

Reply:
we have prefix as an object itself (or package if we have a function). In that case it's like `snap.Storage()`.

Reply:
I am going to resolve this comment, lets's go back to this later if we feel the names are not good.